components/proposalSignForm.tsx

```diff
@@ -202,7 +202,11 @@ function ProposalSignForm({
               ))}
             </div>
             {!!proposal.ui.content.find(
-              v => "addOwners" in v || "removeOwners" in v
+              v =>
+                "addOwners" in v ||
+                "removeOwners" in v ||
+                "changeThreshold" in v ||
+                "adjustExpirationPeriod" in v
```

Comment:
the message doesn't show.
![image](https://github.com/marigold-dev/tzsafe-ui/assets/115862/584da9cc-753c-440a-a250-870df496fa72)